include/constants/snd_def.h

```diff
@@ -0,0 +1,17 @@
+#ifndef POKEHEARTGOLD_SND_DEF_H
+#define POKEHEARTGOLD_SND_DEF_H
+
+#define BGM_VOL_MAX			127							
+#define BGM_VOL_HALF		64							
+#define BGM_VOL_MIN			0								
+#define BGM_VOL_ZUKAN		BGM_VOL_MAX / 3				
+#define BGM_VOL_TR_CASE		BGM_VOL_MAX / 3				
+#define BGM_VOL_CLIP		BGM_VOL_MAX / 3				
+#define BGM_VOL_PERAP		BGM_VOL_MAX / 3				
+#define BGM_VOL_BR			BGM_VOL_MAX / 3				
+#define BGM_VOL_HIROBA_APP	BGM_VOL_MAX / 3				
+
+#define PV_VOL_MAX			127
+#define PV_VOL_DEFAULT		100
+
+#endif //POKEHEARTGOLD_SND_DEF_H
```

Comment:
I would be super careful with taking names directly from Gamefreak's sources and headers.